### PR TITLE
Feat implement send message bird whatsapp msg node

### PR DESCRIPTION
**Pull request by codingsamurai-10**
Sending a custom Whatsapp Message to a given phone number using Messagebird API.
appwrite/appwrite#1891

# Input
1-input

# Successful execution
2-success-execution

# STDOUT
3-stdout

# Whatsapp message received
4-wapp

### CODE REVIEW

File: nodejs/send-message-bird-whatsapp-msg/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  type: "text",
+  content: {
+    text: appwriteParameters.text,
+    disableUrlPreview: false,
+  },
+};
+
+messagebird.conversations.send(params, function (err, response) {
+  if (err) {
+    return console.log(err);
```
Comment (Meldiron) [inline nodejs/send-message-bird-whatsapp-msg/index.js:26]:
```suggestion
    return console.error(err);
```

File: nodejs/send-message-bird-whatsapp-msg/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const sdk = require("node-appwrite");
+require("dotenv").config();
+
+const client = new sdk.Client();
+
+client
+  .setEndpoint(process.env.APPWRITE_API_ENDPOINT) // Your API Endpoint
+  .setProject(process.env.APPWRITE_PROJECT_ID) // Your project ID
+  .setKey(process.env.APPWRITE_API_KEY); // Your secret API key
+
```
Comment (Meldiron) [inline nodejs/send-message-bird-whatsapp-msg/index.js:10]:
```suggestion

```

File: nodejs/send-message-bird-whatsapp-msg/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+A sample Node.js Cloud Function for sending custom Whatsapp Message to a given phone number using Messagebird API.
+
+## 📝 Environment Variables
+
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+
+- **APPWRITE_API_ENDPOINT** - Appwrite API endpoint
+- **APPWRITE_PROJECT_ID** - Appwrite project's ID
+- **APPWRITE_API_KEY** - Appwrite project's API key
```
Comment (Meldiron) [inline nodejs/send-message-bird-whatsapp-msg/README.md:11]:
```suggestion
```

**Review by Meldiron — CHANGES_REQUESTED** [review]
Let's remove unnecessary connection to Appwrite (not used) and also mark the error as.. well, error. (using console.error)

**Review by Meldiron — APPROVED** [review]
Looks good to me

**Comment by Meldiron** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Comment by codingsamurai-10** [comment]
Thank you for making the changes @Meldiron <3

**Pull request merged**